src/scripts/train/OLMo2-1B.py

```diff
@@ -76,48 +73,6 @@ def build_trainer_config(common: CommonComponents) -> TrainerConfig:
                 cancel_check_interval=10,
             ),
         )
-        .with_callback(
```

Comment:
Is this change intentional?

Reply:
Yes. That callback doesn't initialize properly, because some of those datasets don't exist, and none of the other sizes have it.